java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -571,4 +571,41 @@ public void lastsave() {
             assertTrue(Instant.ofEpochSecond(value).isAfter(yesterday));
         }
     }
+
+    @Test
+    @SneakyThrows
+    public void lolwut_lolwut() {
+        var response = clusterClient.lolwut().get();
+        System.out.printf("%nLOLWUT cluster client standard response%n%s%n", response);
```

Comment:
Should we remove the prints? Or do you think we should keep them? Not sure if they're necessary

Reply:
Not necessary, but nobody sees the results without them